src/app/legacy/components/FrostedGlassPromo/index.jsx

```diff
@@ -144,7 +144,7 @@ const FrostedGlassPromo = ({
       ></ClickableArea>
       <ImageWithPlaceholder
         lazyLoad
-        darkMode={isCanonical}
+        darkPlaceholder={isCanonical}
```

Comment:
Instances of `darkPlaceholder` reference to changing the placeholder BBC image from a light to a dark image. This is _not_ linked to the actual dark UI, so this is just a rename of this prop to make it more clear what its actually for.